protected/humhub/modules/user/models/Profile.php

```diff
@@ -157,16 +157,14 @@ private function translationOnly()
         Yii::t('UserModule.profile', 'Gender');
         Yii::t('UserModule.profile', 'Male');
         Yii::t('UserModule.profile', 'Female');
-        Yii::t('UserModule.profile', 'Custom');
+        Yii::t('UserModule.profile', 'Diverse');
         Yii::t('UserModule.profile', 'Hide year in profile');
 
         Yii::t('UserModule.profile', 'Phone Private');
         Yii::t('UserModule.profile', 'Phone Work');
         Yii::t('UserModule.profile', 'Mobile');
         Yii::t('UserModule.profile', 'E-Mail');
         Yii::t('UserModule.profile', 'Fax');
-        Yii::t('UserModule.profile', 'Skype Nickname');
-        Yii::t('UserModule.profile', 'MSN');
         Yii::t('UserModule.profile', 'XMPP Jabber Address');
 
         Yii::t('UserModule.profile', 'Url');
```

Comment:
Personally speaking this should be `Website URL` rather than just `URL`, gives it more meaning and lets the account owner know "Hey promote your website if you have one it's okay!"

Reply:
@ArchBlood Good point. It is also planned to improve the naming too. But this will be another PR.